src/reducers/amountReducer.js

```diff
@@ -0,0 +1,48 @@
+export const initialState = {
+  amountByParties: [],
+  amountByCategories: []
+};
+
+const reducer = (state = initialState, action) => {
+  if (action.type === 'SET_AMOUNT_BY_PARTY') {
```

Comment:
Should this be some "const integer" instead of the whole stringy ?

Reply:
True that. Forgot to change it here